packages/extension/src/newtab/ShortcutLinks/useShortcutLinks.tsx

```diff
@@ -53,12 +56,15 @@ export default function useShortcutLinks(): UseShortcutLinks {
   const shortcutLinks = isTopSiteActive ? sites : customLinks;
   const formLinks = (isManual ? customLinks : sites) || [];
 
+  const { isOldUser, showToggleShortcuts, hasCompletedFirstSession } =
+    useShortcutsUser();
+  const hasNoShortcuts = !shortcutLinks?.length && showTopSites;
+  const showGetStarted =
+    !isOldUser && hasNoShortcuts && !hasCompletedFirstSession;
+
   const resetSelected = () => {
-    if (topSites !== undefined && !hasCustomLinks) {
-      setIsManual(false);
-    } else {
-      setIsManual(true);
-    }
+    const isResetManual = !(topSites !== undefined && !hasCustomLinks);
```

Comment:
All the negations felt a bit too much; maybe this should work. Wdyt?
```suggestion
    const isResetManual = topSites === undefined || !!hasCustomLinks
```